<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Display_Win</class>
 <widget class="QWidget" name="Display_Win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1405</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>451</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Segment</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGraphicsView" name="Segment_view"/>
    </item>
    <item row="1" column="1">
     <widget class="QGraphicsView" name="Depth_view"/>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="Scene_view"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Depth</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Scene</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>841</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Acceleration</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Latitude</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="plotWidget" name="Altitude"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Position</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Altitude</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="plotWidget" name="acceleration"/>
    </item>
    <item row="0" column="1">
     <widget class="plotWidget" name="Latitude"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Pressure</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="plotWidget" name="Longitutde"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Longitude</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="plotWidget" name="Pressure"/>
    </item>
    <item row="0" column="0">
     <widget class="plotWidget" name="Position"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
